### PR TITLE
Syncing script changes from `gh-pages`

### PR DESCRIPTION
**Pull request by paulwallace-okta**

### CODE REVIEW

File: scripts/redirect_url_utils.py [hunk 1]
```diff
@@ -1,14 +1,45 @@
...
-# python3 scripts/redirect_url_utils.py save_map
+"""
+
+Applying existing redirects
+------------------
+
+The default action of this script is to write all existing redirects in
+`file_to_url.txt` to the file system. At present, that's over 8,000 files.
+
+        $ python scripts/redirect_url_utils.py
```
Comment (emikhasyak-okta) [inline scripts/redirect_url_utils.py:11]: Does this instruction work for you? I'm receiving an error
image

File: scripts/redirect_url_utils.py [hunk 2]
```diff
@@ -25,14 +56,34 @@
         <meta http-equiv="REFRESH" content="0.0;url={redirect_to_url}" />
     </head>
     <body></body>
-</html>'''
+</html>
```
Comment (emikhasyak-okta) [inline scripts/redirect_url_utils.py:59]: This line changes every redirect file that we already have in the repository. Can we leave it as it was?
image

we will get rid of all of this later

Comment (paulwallace-okta) [reply scripts/redirect_url_utils.py:59]: sure, reverting

File: scripts/redirect_url_utils.py [hunk 3]
```diff
@@ -28,11 +59,30 @@
...
+      w.write(r)
+
+def _add_redirects_from_file(file):
   total_files = 0
-  with open(FILE_TO_URL, encoding="utf-8") as file_to_url:
+  with open(file, encoding="utf-8") as file_to_url:
     cvs_reader = csv.reader(file_to_url, delimiter=',', quotechar='|')
     for row in cvs_reader:
-      file_path = row[0].lower()
+      file_path = row[0]
```
Comment (emikhasyak-okta) [inline scripts/redirect_url_utils.py:85]: Sorry, this should be the last request. We need `.lower()` when redirects are created paths should be lowercase. (not super critical when all folders are in place, but we can get in trouble when redirects created before we have a content from the build)

**Review by emikhasyak-okta — APPROVED** [review]
Looks good! Thank you for updating those. I left one last request regarding `.lower()` call. 🚢 after addressing

**Pull request merged**